pkg/pxc/app/statefulset/haproxy.go

```diff
@@ -187,6 +187,12 @@ func (c *HAProxy) AppContainer(spec *api.PodSpec, secrets string, cr *api.Percon
 		}
 		appc.Env = append(appc.Env, probsEnvs...)
 	}
+	if cr.CompareVersionWith("1.11.0") >= 0 {
+		appc.VolumeMounts = append(appc.VolumeMounts, corev1.VolumeMount{
+			Name:      "hookscript",
+			MountPath: "/opt/percona/hookscript",
```

Comment:
In the future we can have more than one hook. So, It is better to use `hookscripts`.

Reply:
Also please do the same for logcollector.